xmipp

```diff
@@ -634,6 +642,38 @@ def downloadDeepLearningModels(cwd, dedicatedMode=False):
         ok = pDLdownload.poll() is None or pDLdownload.poll() == 0
     return ok
 
+
+def scipionConnection(dirname):
```

Comment:
methods should resemble verbs (actions), e.g. `linkToScipion`

Reply:
I agree